### PR TITLE
doc: quick start for ratify cli

### PR DESCRIPTION
**Pull request by susanshi**
This PR adds a initial version of cli quickstart doc,  we can improve on it over time. fixes: #36

### CODE REVIEW

**Comment by susanshi** [comment]
I want to discuss what is the expected values for truststore for the cli.

image

File: docs/quickstarts/ratify-cli.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Ratify Cli
```
Comment (FeynmanZhou) [inline docs/quickstarts/ratify-cli.md:1]:
```suggestion
# Ratify CLI
```

Comment (FeynmanZhou) [reply docs/quickstarts/ratify-cli.md:1]: CLI is acronym of **Command Line Interface** so it's recommended to use upper case

File: docs/quickstarts/ratify-cli.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+This document provides a walkthrough on how to use the ratify command line to verify a sample image.
+
+## Verify notation signature with Ratify cli
+
+1. Download the latest Ratify cli
```
Comment (FeynmanZhou) [inline docs/quickstarts/ratify-cli.md:7]:
```suggestion
1. Download the latest Ratify CLI on a Linux AMD64 machine. Go to [release page](https://github.com/ratify-project/ratify/releases/) for other platforms if you are on Windows or Darwin.
```

Comment (FeynmanZhou) [reply docs/quickstarts/ratify-cli.md:7]: It worth mentioning Ratify also supports other platforms not just Linux AMD64.

File: docs/quickstarts/ratify-cli.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+```bash
+curl -L https://github.com/ratify-project/ratify/releases/download/v1.2.1/ratify_1.2.1_Linux_amd64.tar.gz | tar xvzC ~/bin/ ratify
+```
+
+2. Download a local verification certificate for sample image
+```bash
+curl -sSLO https://raw.githubusercontent.com/deislabs/ratify/main/test/testdata/notation.crt
```
Comment (FeynmanZhou) [inline docs/quickstarts/ratify-cli.md:14]:
```suggestion
curl -sSLO https://github.com/ratify-project/ratify/blob/v1.2.1/test/testdata/notation.crt
```

Comment (FeynmanZhou) [reply docs/quickstarts/ratify-cli.md:14]: The documentation is versioned. Should we specify a version for the test certificate?  The test cert in `main` branch might be expired or revoked in the future.

File: docs/quickstarts/ratify-cli.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+                    ]
+                }
+            }
+        ]
+    }
+}
+EOF
+```
+
+Sample cli configuration files for other verifiers can be found in the [plugins](../plugins/verifier/cosign.md#cli) doc.
```
Comment (FeynmanZhou) [inline docs/quickstarts/ratify-cli.md:80]:
```suggestion
Sample CLI configuration files for other verifiers can be found in the [plugins](../plugins/verifier/cosign.md#cli) doc. 
```

**Review by FeynmanZhou — COMMENTED** [review]
Thanks @susanshi . PTAL.

File: docs/quickstarts/ratify-cli.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
+# Ratify CLI
+
+This document provides a walkthrough on how to use the ratify command line to verify a sample image.
+
+## Verify notation signature with Ratify cli
+
+1. Download the latest Ratify CLI on a Linux AMD64 machine. Go to [release page](https://github.com/ratify-project/ratify/releases/) if you are on other platforms.
+```bash
+curl -L https://github.com/ratify-project/ratify/releases/download/v1.2.1/ratify_1.2.1_Linux_amd64.tar.gz | tar xvzC ~/bin/ ratify
```
Comment (binbin-li) [inline docs/quickstarts/ratify-cli.md:9]:
```suggestion
curl -L https://github.com/ratify-project/ratify/releases/download/v1.2.1/ratify_1.2.1_Linux_amd64.tar.gz --output ratify.tar.gz | tar -xvzf ratify.tar.gz -C ~/bin
```

**Review by binbin-li — APPROVED** [review]
lgtm, thanks for the doc!

File: docs/quickstarts/ratify-cli.md (hunk 5 above)
Comment (junczhu) [inline docs/quickstarts/ratify-cli.md:9]: Just curious shall we place the executable under `/usr/local/bin`

File: docs/quickstarts/ratify-cli.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+```
+
+2. Download a local verification certificate for sample image
+```bash
+curl -sSLO https://github.com/ratify-project/ratify/blob/v1.2.1/test/testdata/notation.crt
```
Comment (junczhu) [inline docs/quickstarts/ratify-cli.md:14]: seems we can update the URL to `https://[raw.githubusercontent.com/ratify-project/ratify/dev/test/testdata/notation.crt](https://raw.githubusercontent.com/ratify-project/ratify/dev/test/testdata/notation.crt)`

**Comment by junczhu** [comment]
> I want to discuss what is the expected values for truststore for the cli.
> 
> image

By default, we add `defaultCertsDir` to `VerificationCerts` which leads to this output.
And in k8s scenario, we are using `VerificationCertStores` replacing `VerificationCerts` which avoid this output.

**Comment by binbin-li** [comment]
> what is the expected values for truststore for the cli.

what's the value of `verificationCerts` in your config?

**Comment by susanshi** [comment]
thanks for the review @binbin-li  and @junczhu, i ve pushed some updates, this is ready for another round of review.

**Review by binbin-li — COMMENTED** [review]
lgtm

**Review by FeynmanZhou — APPROVED** [review]
LGTM. Thanks @susanshi

**Review by binbin-li — APPROVED** [review]

**Pull request merged**